analyzers/src/SonarAnalyzer.VisualBasic/Rules/ToStringShouldNotReturnNull.cs

```diff
@@ -42,4 +42,6 @@ public sealed class ToStringShouldNotReturnNull : ToStringShouldNotReturnNullBas
 
     protected override bool IsLocalOrLambda(SyntaxNode node) =>
         node.IsKind(SyntaxKind.MultiLineFunctionLambdaExpression);
+
+    protected override bool IsStatic(SyntaxNode node) => false;
```

Comment:
Actually, for VB you can have "Static" members which are called Shared in VB.NET. You can use [this ](https://converter.telerik.com/)page to convert code samples from C# to VB.NET.